portfolio/src/main/webapp/index.html

```diff
@@ -2,45 +2,77 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Austin Teshuba</title>
-    <link rel="stylesheet" href="style.css">
+    <title>
+      Austin Teshuba
+    </title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;500;700&display=swap" rel="stylesheet">
```

Comment:
Is there documentation for this API? You may want to leave a link in an HTML comment.

Reply:
It's actually just an embedded font from Google Fonts. Adding comment with link to site for documentation